packages/global-styles/src/components/global-styles-variations/preview.tsx

```diff
@@ -133,7 +133,7 @@ const GlobalStylesVariationPreview = ( { title, inlineCss, isFocused, onFocusOut
 										style={ {
 											height: normalizedColorSwatchSize * ratio,
 											width: normalizedColorSwatchSize * ratio,
-											background: color,
+											background: color.color,
```

Comment:
@arthur791004, The types here looked a bit off -- is this what the code is meant to do? It seems that `style` wants the `background` to be a different type, but the color object has three properties.

Reply:
The type definition is incorrect here as the `highlightedColors` is `Color[]`. Hence, the type should be

```diff
- ( { slug, color }: { slug: string; color: Color }, index: number ) => (
+ ( { slug, color }: { slug: string; color: string }, index: number ) => (
```

Besides, it might be better to rename the type definition from `Color` to something like `ColorPalette` since the `Color` might be too general and make people misunderstand the meaning 😅

Reply:
fixed